plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchMachineTypeSelector.groovy

```diff
@@ -71,6 +71,8 @@ class GoogleBatchMachineTypeSelector {
      */
     private static final List<String> DEFAULT_FAMILIES_WITH_SSD = ['n1-*', 'n2-*', 'n2d-*', 'c2-*', 'c2d-*', 'm3-*']
 
+    private static final List<String> DEFAULT_FAMILIES_WITH_HDD = ['n1-*', 'n2-*', 'n2d-*', 'c2-*', 'c2d-*', 'm1-*', 'e2-*']
```

Comment:
My only doubt here if we also add `m2` and `m3` that are types with very big memories
https://cloud.google.com/compute/docs/memory-optimized-machines#m2_machine_types

Reply:
Also these types are not "HDD" types. The difference is that there are some types that allow to mount a local SSD disk and others that only allow network disks. So maybe better naming can be here `DEFAULT_FAMILIES` and for Fusion `DEFAULT_FAMILIES_WITH_LOCAL_SSD`